kroxylicious-api/src/main/java/io/kroxylicious/proxy/ApiMaturity.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
I stole these annotations from Apache Kafka, but have changed them slightly (with the aim of better suiting our needs). I don't think it's entirely right to assert copyright over them, even though I understand that such copyright assertions aren't necessarily meaningful from a (US) legal PoV.

Reply:
Given the build fails without us asserting copyright it seems necessary to hold your nose and do so.